### PR TITLE
Add documentation for the LDAP auth module

### PR DESCRIPTION
**Pull request by brodo**
This PR adds documentation for the [LDAP auth module](https://github.com/parse-community/parse-server/pull/6226).

### CODE REVIEW

File: _includes/parse-server/third-party-auth.md
Comment (TomWFox) [inline _includes/parse-server/third-party-auth.md]:
```suggestion
To build a query which works with your LDAP server, you can use a LDAP client like [Apache Directory Studio](https://directory.apache.org/studio/).
```

File: _includes/parse-server/third-party-auth.md
Comment (dplewis) [inline _includes/parse-server/third-party-auth.md]: Extra line break

File: _includes/parse-server/third-party-auth.md
Comment (dplewis) [inline _includes/parse-server/third-party-auth.md]: {{id}} is replaced with the id supplied in authData.

File: _includes/parse-server/third-party-auth.md
Comment (dplewis) [inline _includes/parse-server/third-party-auth.md]: Extra line breaks

**Review by dplewis — CHANGES_REQUESTED** [review]

File: _includes/parse-server/third-party-auth.md
Comment (TomWFox) [reply _includes/parse-server/third-party-auth.md]: Unresolved as the line breaks are still present 🙂

**Review by TomWFox — APPROVED** [review]
Looks good to me! As standard we’ll wait until the server pr is merged and a release is published before merging the docs pr.

**Comment by brodo** [comment]
The PR in the other repo is merged now.

**Comment by TomWFox** [comment]
👍 I’ll wait for a release to merge this.

**Comment by TomWFox** [comment]
Merging as 3.10 has been released.

**Comment by TomWFox** [comment]
@brodo could you resolve the merge conflict and then we'll get this in

**Review by dplewis — APPROVED** [review]

**Comment by TomWFox** [comment]
Thanks @dplewis :)

**Pull request merged**
